packages/backend/src/connector/core/middleware/error-handler.ts

```diff
@@ -26,7 +26,15 @@ export function createIncomingErrorHandlerMiddleware(
       if (ctx.revertTotalReceived) {
         await ctx.revertTotalReceived()
       }
-      ctx.response.reject = errorToIlpReject(serverAddress, err)
+      if (isIlpError(err)) {
+        ctx.response.reject = errorToIlpReject(serverAddress, err)
+      } else {
+        ctx.response.reject = errorToIlpReject(serverAddress, {
+          message: err && err['message'],
+          ilpErrorCode: IlpErrorCode.F00_BAD_REQUEST,
```

Comment:
what about
```suggestion
          message: 'unexpected internal error',
          ilpErrorCode: IlpErrorCode.T00_INTERNAL_ERROR,
```
but log `err && err['message']`?

Reply:
Good idea